### PR TITLE
adds rot13labs rotc0n macropad

### PR DESCRIPTION
**Pull request by c0ldbru**
## Description

Adds rot13labs rotc0n macropad and keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rot13labs/rotc0n/config.h
Comment (zvecr) [inline keyboards/rot13labs/rotc0n/config.h:1]: This file should be removed.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/rot13labs/rotc0n/info.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "rows": ["B1", "B0"]
+    },
+    "processor": "atmega328p",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0xBEEF",
+        "vid": "0xFEED"
+    },
+    "layouts": {
```
Comment (zvecr) [inline keyboards/rot13labs/rotc0n/info.json:26]:
```suggestion
    "community_layouts": ["ortho_2x3"],
    "layouts": {
```

File: keyboards/rot13labs/rotc0n/config.h
Comment (c0ldbru) [reply keyboards/rot13labs/rotc0n/config.h:1]: nuked!

**Comment by c0ldbru** [comment]
Just checking in on this to see if theres anything else I can do to help move this PR forward?

File: keyboards/rot13labs/rotc0n/readme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# rotcon
+
+![rotcon](https://i.imgur.com/InAAdPp.jpg)
```
Comment (fauxpark) [inline keyboards/rot13labs/rotc0n/readme.md:3]:
```suggestion
![rotcon](https://i.imgur.com/InAAdPph.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by c0ldbru** [comment]
Anything else needed for this PR?

**Review by fauxpark — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**